docs/xxjitserveraotcachepersistence.md

```diff
@@ -0,0 +1,57 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:[+|-]JITServerAOTCachePersistence
+
+**(Linux&reg; only)**
+
+This option enables or disables the JITServer server's ability to allow its later instances to reuse AOT caches.
+
+## Syntax
+
+        -XX:[+|-]JITServerAOTCachePersistence
+
+| Setting                    | Effect  | Default                                                                              |
+|----------------------------|---------|:------------------------------------------------------------------------------------:|
+|`-XX:+JITServerAOTCachePersistence` | Enable  |                                                                                      |
+|`-XX:-JITServerAOTCachePersistence` | Disable | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+
+## Explanation
+
+ With the `-XX:+JITServerAOTCachePersistence` option, the JITServer server periodically saves its AOT caches to files, and then loads these caches from files the first time a client requests a particular named cache. It is useful to improve performance, especially in scenarios, where JITServer instances are started up and shut down regularly.
```

Comment:
Current phrasing suggests that the same JITServer instance saves a cache and then loads it. It's another JITServer instance that may load a saved AOT cache.
Maybe we should rephrase:
"With the `-XX:+JITServerAOTCachePersistence` option, the JITServer server periodically saves its AOT caches to files, allowing other JITServer instances to loads these caches from files the first time a client requests a particular named cache."